src/blockchain/utils.ts

```diff
@@ -55,12 +55,12 @@ export const fixedSpecial = (num, n) => {
 
 export const BNtoNum = (value, decimal = 18) => {
```

Comment:
type of function params, type of return value is not defined

Reply:
done